src/pages/use-cases/service-mesh.jsx

```diff
@@ -56,6 +56,14 @@ const sectionContent4 = {
   imageRight: false,
 };
 
+const sectionContent5 = {
+  title: 'Hands-On: eCHO Playlist',
+  videoSrc:
+    'https://www.youtube.com/embed/videoseries?si=quRWHYILz6cH2kmO&amp;list=PLDg_GiBbAx-l5U7CdEHtg1DJL2kkOlPTE',
```

Comment:
can you reverse the order of the playlist so the introduction is first and it builds from there?

Reply:
Sure, it's updated.
https://www.youtube.com/playlist?list=PLDg_GiBbAx-l5U7CdEHtg1DJL2kkOlPTE